### PR TITLE
Add Go 1.10 and 1.11 to Travis

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ sudo: false
 
 go:
   - 1.9
+  - 1.10
```
Comment (jordan-wright) [inline .travis.yml]:
```suggestion
  - "1.10"
```

**Comment by jordan-wright** [comment]
Thanks for sending this in! It looks like Travis CI is parsing the 1.10 as 1.1, which is related to [this bug](https://github.com/travis-ci/travis-ci/issues/9247). I've added a suggestion that should fix the issue if you want to double-check it 😄 

Thanks again for this!

**Comment by gliptak** [comment]
@jordan-wright thank you for the pointer. change applied

**Comment by jordan-wright** [comment]
LGTM! Thanks for the quick fix.

**Pull request merged**